.travis.yml

```diff
@@ -7,18 +7,13 @@ services:
 - docker
 install:
 - pip install --no-cache-dir -r dev-requirements.txt
-- . ci/minikube.env
-- ./ci/install.sh
```

Comment:
What was your thinking behind moving everything into `script` instead of running the install/setup things here? Might be worth writing a comment for future generations.